backend/migrations/script.py.mako

```diff
@@ -0,0 +1,24 @@
+"""${message}
```

Comment:
These files are already present in the api folder. __Duplicate__ it is. @vaibhavsingh97